### PR TITLE
Add more executables to submodule

### PR DESCRIPTION
**Pull request by johnnie502**
Add `gsed`, `docker_host_user_id.sh` and `install_composer` to the submodule since it was only used in STDcheck, and now it will be used by other projects.
From code review change request in
https://github.com/Medology/stdcheck.com-client/pull/1#discussion_r542435586
https://github.com/Medology/stdcheck.com-client/pull/1#discussion_r542452365
https://github.com/Medology/stdcheck.com-client/pull/1#discussion_r542541102

### CODE REVIEW

File: bin/install_composer [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+curl -L https://getcomposer.org/download/${EXPECTED_VERSION}/composer.phar > "$ROOT"/bin/composer
+
+echo -n "Checking the hash..."
+if [[ ! $(shasum -a 384 "${ROOT}"/bin/composer) = "$EXPECTED_CHECKSUM"* ]]; then
+    echo "ERROR: composer's sha384 doesn't match. Redownload it."
+    rm "${ROOT}"/bin/composer
+    exit 1
+else
+    echo "passed"
+fi
```
Comment (rashidsaf) [inline bin/install_composer:44]: I don't think else is necessary here, since we have `exit`

File: bin/install_composer [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+EXPECTED_VERSION=2.0.8
+EXPECTED_CHECKSUM=608432e9a850f02b527a661e6771546a10a8cc16e17363754c101f06d1ab03e53106d7aef2b5faa606d67523fcf33dc3
+
+# Check for existing installation
+if [ -e "$ROOT"/bin/composer ]; then
+    echo "Checking Composer version..."
+    FOUND_VERSION=$( \
+        cd "$ROOT" && \
+        ./bin/composer --version | \
```
Comment (rashidsaf) [inline bin/install_composer:15]: I feel like these 2 lines should be in the same line to make it more readable, as each pipe will be in its own

**Review by rashidsaf — APPROVED** [review]
Just some optional cosmetic changes

**Review by fortoulrodolfo — APPROVED** [review]

File: bin/install_composer [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    echo "ERROR: composer's sha384 doesn't match. Redownload it."
+    rm "${ROOT}"/bin/composer
+    exit 1
+else
+    echo "passed"
+fi
+
+# Fix permissions
+echo "Setting composer permissions..."
+chmod 755 "$ROOT"/bin/composer
```
Comment (BKorenek) [inline bin/install_composer:48]: is 755 not a little extreme here? 🤷

**Review by BKorenek — APPROVED** [review]

File: bin/install_composer (hunk 3 above)
Comment (johnnie502) [reply bin/install_composer:48]: 755 is the default for all executable in submodule 🤔

**Pull request merged**